spec/fs-plus-spec.coffee

```diff
@@ -516,6 +516,9 @@ describe "fs", ->
         expect(fs.tildify(fixture)).toBe fixture
         expect(fs.tildify('foo')).toBe 'foo'
 
+    it "should not tildify URIs", ->
+      uri = 'remote://server:123/folder/remote_file.txt'
+      expect(fs.tildify(uri)).toBe uri
```

Comment:
I would like to spy on the tildify module to ensure it isn't called but I don't know if there is a easy way to spy on a required module in jasmine.